app/javascript/lib/MessageModal.js

```diff
@@ -15,6 +15,24 @@ export function showMessage(message, key, logProps) {
   functionHolder.showMessage(message, key, logProps)
 }
 
+/** take a json-api error object and render a modal with the contents.
+ * For now, this just uses the 'detail' field of each error.
```

Comment:
The formal spec is nice and readable; linking it here would be convenient reference.

```suggestion
 * For now, this just uses the 'detail' field of each error.
 * Details on JSON API error object format: https://jsonapi.org/format/#error-objects
```